src/DeviceInput/InputDevices/deviceSource.ts

```diff
@@ -5,7 +5,7 @@ import { IDeviceInputSystem } from './inputInterfaces';
 import { IUIEvent } from '../../Events/deviceInputEvents';
 
 /** @hidden */

```

Comment:
nit: this is useless if it's not exported.

Reply:
Removed hidden